script/core

```diff
@@ -40,6 +40,7 @@ if [ -n "$ref" ]; then
   echo "Installing Home Assistant core at ${ref}..."
   python3 -m pip install --user --upgrade --src "$HOME/src" \
   --editable "git+${coreURL}@${ref}#egg=homeassistant"
+  (cd "$HOME/src/homeassistant" && python3 -m script.translations develop --all)
```

Comment:
1.  We should make this contingent on the existence of the frontend translations since it's perfectly valid to develop in English only.  Put an if block to check for `translations/token.json`.
2. Do we need to `cd` or can we just reference the full path?  If needed, then `cd -` after.
3. Lose the `&&` and use multiple lines since we're already in a script

Reply:
I'm not sure 1. means and haven't found anywhere a file called token.json in the translations folder. 
And for 2 en 3, cd is required and I think the one-liner is prettier and easier to maintain?

Reply:
> I'm not sure 1. means and haven't found anywhere a file called token.json in the translations folder. Could you explain a bit more?

If you don't have that file, then you aren't pulling the frontend translations.  If you aren't pulling them for the frontend, then why bother pulling them for the backend?

> And for 2 and 3, cd is required and I think the one-liner is prettier and easier to maintain?

Condensing things into 1 line is Easier to maintain? 😕

Reply:
> But going further on point 1: I have a setup in a codespace with the frontend translations being present and some strange "artifact.json" file, but when I do "Develop HA in a devcontainer"-task along with a "Developer Frontend"-task, backend translations aren't there. Starting config flows just show the form fields with no textes, creating a helper group, just shows an empty list. So, that's why I suggest in this PR to run that script, which will just generate English backend translations without pulling anything from lokalize.


I forgot the core script is actually setting up English, and I wasn't paying attention enough and missed the subshell - keep as is and just avoid the extra process:


```suggestion
  (cd ~/src/homeassistant && exec python3 -m script.translations develop --all)
```

 
Also, if you are missing `token.json`, you aren't updating the frontend translations.  The artifact file is just the meta from the last run you downloaded them from on GitHub, but without a token the `fetch-nightly-translations` gulp task will just skip and setup English only.  If you want other languages, rerun the setup task to create another token.